js/main.js

```diff
@@ -25,5 +25,81 @@ $form.addEventListener('submit', function (event) {
 
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
 
+  $entryformDiv.className = 'hidden';
+  $entriesDiv.className = 'entries';
+  data.view = 'entries';
   $form.reset();
+  window.location.reload();
+});
+
+function treeMaker(entry) {
+
+  var ulElement = document.createElement('ul');
+  ulElement.setAttribute('class', 'entry-list');
+
+  var liImg = document.createElement('li');
+  liImg.setAttribute('class', 'entry-item');
+  ulElement.appendChild(liImg);
+
+  var imgTag = document.createElement('img');
+  imgTag.setAttribute('class', 'img-entry');
+  imgTag.setAttribute('src', entry.src);
+  liImg.appendChild(imgTag);
+
+  var columnRight = document.createElement('div');
+  columnRight.setAttribute('class', 'entry-column');
+  ulElement.appendChild(columnRight);
+
+  var liTitle = document.createElement('li');
+  liTitle.setAttribute('class', 'entry-item');
+  columnRight.appendChild(liTitle);
+
+  var divTitle = document.createElement('div');
+  divTitle.setAttribute('class', 'title-entry');
+  divTitle.textContent = entry.title;
+  liTitle.appendChild(divTitle);
+
+  var liDescription = document.createElement('li');
+  liDescription.setAttribute('class', 'entry-item');
+  columnRight.appendChild(liDescription);
+
+  var divDescription = document.createElement('div');
+  divDescription.setAttribute('class', 'description-entry');
+  divDescription.textContent = entry.textArea;
+  liDescription.appendChild(divDescription);
+
+  return ulElement;
+}
+
+var $entriesList = document.querySelector('[data-view="entries"]');
+
+window.addEventListener('DOMContentLoaded', function (event) {
+  for (var i = 0; i < data.entries.length; i++) {
+    var tree = treeMaker(data.entries[i]);
+    $entriesList.appendChild(tree);
```

Comment:
This will need to be revised after your revise the function not to create a `ul`